### PR TITLE
feature/validation2

### PR DESCRIPTION
**Pull request by Mechakhan**
Improve ValidationService by not relying on separate HTTP requests to obtain the certificates and add more detailed error messages.

### CODE REVIEW

**Comment by Mechakhan** [comment]
@Cactu5 This should be ready for review. After looking through it, I would like your opinion on what to do about ValidationServiceTest.java, as those tests don't really fit the new ValidationServices. Should they be rewritten to instead call the services using different Certificates and ISO 20022 messages or should they be removed since SendPacs008Tests.java indirectly tests the ValidationServices?

File: decentralizediso20022/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,7 +1,26 @@
...
-message.bad-pacs=The entity was not a valid pacs message.
-message.500=Could not process the message.
-message.200=The message was processed.
\ No newline at end of file
+message.empty=The message was empty.
+message.bad-pacs=The message was not a valid pacs message.
+
+message.bad-from-header=The message's From tag "%s" did not match the certificate's common name "%s".
+message.bad-to-header=The message's To tag "%s" did not match the certificate's common name "%s".
+message.bad-recipient=Failed to retrieve the recipient from the message's To tag "%s".
```
Comment (Cactu5) [inline decentralizediso20022/src/main/resources/application.properties:8]: This file has weird line endings, could this be fixed before merging?

Comment (Mechakhan) [reply decentralizediso20022/src/main/resources/application.properties]: Has been fixed

**Review by Cactu5 — APPROVED** [review]
👍

**Comment by Mechakhan** [comment]
> 👍

Great! Please merge if you're OK with the ValidationServiceTests being commented out.

**Pull request merged**